chromium_src/chrome/browser/history/history_utils.cc

```diff
@@ -7,9 +7,22 @@
 #include "../../../../../chrome/browser/history/history_utils.cc"  // NOLINT
 #undef CanAddURLToHistory
 
+#if defined(ENABLE_EXTENSION)
```

Comment:
please add `#include "extensions/buildflags/buildflags.h"` so aren't just getting it by luck through some header chain in `history_utils.cc`

Reply:
Oops, as I said in https://github.com/brave/brave-core/pull/3527#discussion_r328415482,
it should be `#if BUILDFLAG(ENABLE_EXTENSION)` not `#if defined(...)`

Reply:
done